deploy/kubernetes/images/dev/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+namespace:
+  gce-pd-csi-driver
+resources:
+- image.yaml
+- ../alpha/
```

Comment:
If we ever need to change the tag for node-driver-registrar in the dev overlay, should it use the image name in base or alpha?

Reply:
if it is only for dev, then change the image in the image.yaml file under /dev

Reply:
there is no image name gke.gcr.io/gcp-compute-persistent-disk-csi-driver-win (this is for windows) in base (stable). 
Generally, the following can happen
in stable, change image name --> nameA
in alpha, change image name --> nameB
if alpha is also based on stable, I think the final image name will be nameA.
Or,
in stable, change image name --> nameA
in alpha, change image nameA --> nameB
if alpha is also based on stable, I think the final image name will be nameB.

This is thought ImageTagTransformer. I also tested using both ImageTagTransformer and images section directly in kustomization.yaml. The latter (images section) seems taking the final effect.
There are so many ways to construct this, we just have to try it out.